test/toeplitz/test_apply_interpolation.py

```diff
@@ -0,0 +1,23 @@
+import torch
+from gpytorch.kernels import RBFKernel, GridInterpolationKernel
+from gpytorch import utils
+from gpytorch.utils.toeplitz import index_coef_to_sparse
+from torch.autograd import Variable
+
+
+def test_apply_interpolation():
```

Comment:
This should belong in the new `test/utils/toeplitz_test.py` file.
The convention for test files is: replace `gpytorch` with `test`, and replace `.py` with `_test.py`.
I'm not even sure pytest is picking up on this file, since it is picky about naming conventions.